Fix:

error: implicit declaration of function 'sleep' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'unlink' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

https://github.com/phmarek/fsvs/issues/24
--- src/add_unvers.c.orig	2023-12-13 06:06:55.000000000 -0600
+++ src/add_unvers.c	2024-07-18 00:00:35.000000000 -0500
@@ -7,6 +7,7 @@
  ************************************************************************/
 
 #include <fcntl.h>
+#include <unistd.h>
 
 #include "global.h"
 #include "add_unvers.h"
--- src/cp_mv.c.orig	2023-12-13 06:06:55.000000000 -0600
+++ src/cp_mv.c	2024-07-18 00:02:28.000000000 -0500
@@ -8,6 +8,7 @@
 
 #include <fcntl.h>
 #include <search.h>
+#include <unistd.h>
 
 #include "global.h"
 #include "cp_mv.h"
--- src/hash_ops.c.orig	2023-12-13 06:06:55.000000000 -0600
+++ src/hash_ops.c	2024-07-18 00:07:31.000000000 -0500
@@ -9,6 +9,7 @@
 #include <fcntl.h>
 #include <errno.h>
 #include <string.h>
+#include <unistd.h>
 
 #include "global.h"
 #include "waa.h"
--- src/resolve.c.orig	2023-12-13 06:06:55.000000000 -0600
+++ src/resolve.c	2024-07-18 00:10:18.000000000 -0500
@@ -9,6 +9,7 @@
 #include <fcntl.h>
 #include <sys/uio.h>
 #include <sys/mman.h>
+#include <unistd.h>
 
 
 #include "global.h"
